eq-author/src/constants/validationMessages.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  ERR_VALID_REQUIRED: ({ label }) => `${label} is required`,
```

Comment:
It would be nice to share this constant name between client and server. But I don't think we have a nice way of doing this yet.